### ISSUE TITLE
l'estensione non funziona

### ISSUE THREAD

**Issue by zizzi65** [vsDizzy/SaveAsMHT#13]
L'estensione a smesso di funzionare su tutti i siti improvvisamente senza un motivo apparente provato a rimuoverla e reinstallare ma estensione non da piu' segni di vita

**Comment by Arickx** [comment]
Today I also noticed I was unable to save web pages in '.mht' format with this 'Save As MHT' extension :(
I should also like to know why and if there is a solution or a good alternative?
I added now, for the moment, the Archiver extension ( https://github.com/rahiel/archiveror ) to save web pages in '.mhtml' format. By changing the filename '°.mhtml' to '°.mht', I am still able to see easily the original page information with the 'Save As MHT' extension. I am missing some options in the Archiver extension:
- save as '.mht' with right mouse click;
- original page information.
I am looking forward for a better solution.
